README.md

```diff
@@ -872,6 +882,16 @@ const inputParams = serializeUpdateContractParameters(
         modulefileBuffer
 );
 ```
+Note that if using a V1 contract, the 5th argument to `serializeUpdateContractParameters` should be 1:
```

Comment:
We are using semantic versioning for the schemas (although we only used the major version so far), so while this is fine atm. it won't be in the future.
We decided that schema v3 will include the version directly in the schema (as opposed to only in the wasm module). That will alleviate some of the problems you are facing here.
Although v1 and v2 schemas should still be supported by our tools in the future.